### PR TITLE
[TAS-918] 🚸 Add iscnOwner to stakeholders

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2024-01-09 晚上8 26 35
截圖 2024-01-09 晚上8 26 45

### CODE REVIEW

File: components/IscnRegisterForm.vue [hunk 1]
```diff
@@ -1109,21 +1114,38 @@ export default class IscnRegisterForm extends Vue {
...
-    this.uploadStatus = 'loading'
-    // ISCN Fee needs Arweave fee to calculate
-    await this.calculateISCNFee()
-    this.uploadStatus = ''
   }
+  if (this.address) {
+    const iscnOwner = {
+      name: this.$t('iscn.meta.stakeholders.name.placeholder') as string,
+      wallet: [{
+        content: 'like1qv66yzpgg9f8w46zj7gkuk9wd2nrpqmca3huxf',
```
Comment (williamchong) [inline components/IscnRegisterForm.vue:1133]:
```suggestion
        content: this.address,
```

[the remark this answers is not in the document]
Comment (AuroraHuang22) [reply components/IscnRegisterForm.vue]: 😅 updated

File: components/IscnRegisterForm.vue [hunk 2]
```diff
@@ -809,7 +809,7 @@ export default class IscnRegisterForm extends Vue {
   thumbnailUrl: string = ''
   authorName: string = ''
   authorUrl: string[] = []
-  authorWalletAddress: string[] = []
+  authorWalletAddress: any = []
```
Comment (williamchong) [inline components/IscnRegisterForm.vue:812]:
```suggestion
  authorWalletAddress: any[] = []
```

File: types/author.d.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export interface Author {
   name: string;
   url: string[];
-  wallet: string[];
+  wallet: any;
```
Comment (williamchong) [inline types/author.d.ts:4]:
```suggestion
  wallet: any[];
```

**Review by williamchong — CHANGES_REQUESTED** [review]
Please keep array type if wallet must be array of object

File: components/IscnRegisterForm.vue (hunk 2 above)
Comment (AuroraHuang22) [reply components/IscnRegisterForm.vue]: [4d26903](https://github.com/likecoin/app-like-co/pull/435/commits/4d26903c8f4d854ec1fbc8866d04ea7777122717)

**Review by williamchong — APPROVED** [review]

**Pull request merged**